### ISSUE TITLE
Resnet18的layer中存在未二值化的卷积

### ISSUE THREAD

**Issue by yuhua666** [ZiweiWangTHU/BiDet#53]
`
def _make_layer(self, block, planes, blocks, stride=1, **kwargs):
        downsample = None
        if stride != 1 or self.inplanes != planes * block.expansion:
            conv = nn.Conv2d
            ds_out_planes = planes * block.expansion
            downsample = nn.Sequential(
                nn.AvgPool2d(2, stride=stride, ceil_mode=True),
                conv(self.inplanes, ds_out_planes, kernel_size=1, stride=1, bias=False),
                nn.BatchNorm2d(ds_out_planes)
            )

        layers = []
        layers.append(block(self.inplanes, planes, stride, downsample, **kwargs))
        self.inplanes = planes * block.expansion
        for _ in range(1, blocks):
            layers.append(block(self.inplanes, planes, **kwargs))

        return nn.Sequential(*layers)
`

博士您好，如上面这段代码，的第四行，该卷积未被二值化，想请教一下这是为什么呢？

**Comment by Wuziyi616** [comment]
你好，因为我们发现对于降采样的 Conv, 二值化对性能影响很大，而且这只是一个 1x1 Conv, 本身 size/计算量就很小，因此牺牲这一点计算量可以换取不小的性能提升

**Comment by yuhua666** [comment]
感谢！
